src/go/k8s/.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/vectorizedio/pre-commit-vectorizedio.git
```

Comment:
who runs this?

Reply:
@ivotron FYI

Reply:
I created the pre-commit hooks to never commit invalid code (test and linters should always pass). My intention is to short the time between running the linters and create commits. One could push the changes to the remote and wait for github action. I feel this is wait of time. 

The `crlfmt` hooks was created, because I didn't find it in opensource. The `golangci-lint` was modified to work correctly in `redpanda` repo as the code lives in `src/go/k8s`.